src/components/BuildBaronAndAnnotations/AnnotationTickets.tsx

```diff
@@ -54,22 +54,26 @@ export const AnnotationTickets: React.FC<Props> = ({
           </StyledButton>
         </ConditionalWrapper>
       </TitleAndButtons>
-      <AnnotationTicketsTable
-        jiraIssues={tickets}
-        annotationId={annotationId}
-        taskId={taskId}
-        execution={execution}
-        isIssue={isIssue}
-        userCanModify={userCanModify}
-      />
-      <AddIssueModal
-        dataCy="addIssueModal"
-        visible={isAddAnnotationModalVisible}
-        closeModal={() => setIsAddAnnotationModalVisible(false)}
-        taskId={taskId}
-        execution={execution}
-        isIssue={isIssue}
-      />
+      {tickets?.length > 0 && (
+        <>
+          <AnnotationTicketsTable
+            jiraIssues={tickets}
+            annotationId={annotationId}
+            taskId={taskId}
+            execution={execution}
+            isIssue={isIssue}
+            userCanModify={userCanModify}
+          />
+          <AddIssueModal
```

Comment:
Did you test it to make sure that the AddIssueModal shows up when the button is clicked? I'm not sure how this being hidden behind a `tickets?.length > 0` conditional affects it.

Reply:
Good point, this should just cover the table. It's fixed now.